packages/strapi-helper-plugin/lib/src/components/Select/styles.js

```diff
@@ -101,6 +101,7 @@ const getStyles = theme => {
       ...base,
       padding: '2px 10px 4px 10px', // These value don't exist in the theme
       fontSize: sizes.fonts.md,
+      lineHeight: '18px',
```

Comment:
This one might break the UI other dropdowns nope?

Reply:
Nope I have tested it and it's working.